db/views/national_hearing_queue_entries_v01.sql

```diff
@@ -0,0 +1,12 @@
+--- this view is a way to see both of the appeals and legacy appeals information in a single table
+--- materialized means that this information will be cached in a temporary table
+SELECT
+    appeals.id AS appeal_id,
+    'appeal' as appeal_type,
```

Comment:
```suggestion
    'Appeal' as appeal_type,
```
We'll want type columns to match the same capitalization as Ruby class names through this effort to make some of our work later easier.